network-api/networkapi/templates/fragments/svg/nav_icons.html

```diff
@@ -0,0 +1,5 @@
+{% if icon_type == "external_link" %}
+
+    <svg class="tw-inline-block tw-stroke-gray-20 group-hover/item:tw-stroke-blue-80" width="18" height="18" fill="none" xmlns="http://www.w3.org/2000/svg"><g stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"><path d="M5.25 12.75l7.5-7.5M5.25 5.25h7.5v7.5"/></g></svg>
```

Comment:
Using inline SVG so we can smoothly transition the SVG from default state to hover state using CSS